topo/node/cisco/cisco.go

```diff
@@ -148,18 +156,100 @@ func (n *Node) Create(ctx context.Context) error {
 		return fmt.Errorf("failed to create pod for %q: %w", pb.Name, err)
 	}
 	log.Debugf("Pod created:\n%+v\n", sPod)
-	log.Infof("Created XRD node resource %s pod", n.Name())
+	log.Infof("Created Cisco %s node resource %s pod", n.Proto.Model, n.Name())
 	if err := n.CreateService(ctx); err != nil {
 		return err
 	}
-	log.Infof("Created XRD node resource %s services", n.Name())
+	log.Infof("Created Cisco %s node resource %s services", n.Proto.Model, n.Name())
 	return nil
 }
 
+func constraints(pb *tpb.Node) map[string]string {
+	constraints := map[string]string{
+		"cpu":    "1",
+		"memory": "2Gi",
+	}
+	switch pb.Model {
+	case "8201", "8201-32FH", "8202", "8101-32H", "8102-64H":
+		constraints = map[string]string{
+			"cpu":    "4",
+			"memory": "12Gi",
+		}
+	case "xrd":
+		constraints = map[string]string{
+			"cpu":    "1",
+			"memory": "2Gi",
+		}
+	}
+	return constraints
+}
+
+func generateInterfacesEnvs(pb *tpb.Node) (interfaceList, interfaceMap string) {
+	interfaceList = "MgmtEther0/RP0/CPU0/0"
+	interfaceMap = "MgmtEther0/RP0/CPU0/0:eth0"
+	eths := make([]string, 0)
+	for k, _ := range pb.Interfaces {
+		eths = append(eths, k)
+	}
+	sort.Strings(eths)
+	for _, eth := range eths {
+		if !strings.HasPrefix(eth, "eth") {
```

Comment:
so what happens to these interfaces? is there an example of them not matching "eth" ?

Reply:
e8000 containers provide data plane functionalities and matches production image requirement.

Reply:
@marcushines  I need your advice for this case. I did a quick test and it seems that cluster bring up (networkop/init-wait) fails when the interface id is set to anything other than [e|et|eth|...]+[id]. So, I think there is a need to have a  check on KNE to prevent users from setting non-valid ethernet id. However, for this PR, If you agree, I will only handle the interface id in the correct format and logs a warning otherwise.

Reply:
code is changed to return error and fail early. Add ut to check this case as well.